Gems/EMotionFX/Code/Source/Editor/SkeletonModel.cpp

```diff
@@ -618,7 +656,7 @@ namespace EMotionFX
         const size_t numLodLevels = actor->GetNumLODLevels();
         const Skeleton* skeleton = actor->GetSkeleton();
         const size_t numNodes = skeleton->GetNumNodes();
-        m_nodeInfos.resize(numNodes);
+        m_nodeInfos.resize(numNodes + 1);
```

Comment:
Can you add some comments on the places where we do index manipulations like +1 and -1 please?